db/migrate/20220603083700_create_posts.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.bigint :author_id, index: true
+      t.string :title
+      t.string :text
```

Comment:
- [ ] [OPTIONAL] I think it would be better to use text type instead of string for the text because the text(description) may be longer